qemu/tests/disk_extension.py

```diff
@@ -69,6 +69,10 @@ def update_loop_device_backend_size(backend_img, device_img, size):
     size_unit = params["increment_size"][-1]
     guest_cmd = params["guest_cmd"]
 
+    loop_device = process.run("losetup -f").stdout.decode().strip()
+    params["image_name_stg1"] = loop_device
+    params["loop_device"] = loop_device
```

Comment:
Please remove the `loop_device = /dev/loop6` and `image_name_stg1 = ${loop_device}` from cfg since you have set it dynamically.